attributes/default.rb

```diff
@@ -238,26 +245,55 @@
   'default' => "659400f9842fffb8d64ae0b650f081b9",
   'ubuntu' => { 'default' => "c2b410031867dc7c966ca5b1aa0c72aa" }
 )
+
+# MySQL Packages
+default['cluster']['mysql']['package']['root'] = "#{node['cluster']['artifacts_s3_url']}/mysql"
+default['cluster']['mysql']['package']['version'] = "8.0.31-1"
+default['cluster']['mysql']['package']['source-version'] = "8.0.31"
+default['cluster']['mysql']['package']['platform'] = if arm_instance?
+                                                       value_for_platform(
+                                                         'default' => "el/7/aarch64"
+                                                       )
+                                                     else
+                                                       value_for_platform(
+                                                         'default' => "el/7/x86_64",
+                                                         'ubuntu' => {
+                                                           '20.04' => "ubuntu/20.04/x86_64",
+                                                           '18.04' => "ubuntu/18.04/x86_64",
+                                                         }
+                                                       )
+                                                     end
+default['cluster']['mysql']['package']['file-name'] = "mysql-community-client-#{node['cluster']['mysql']['package']['version']}.tar.gz"
+default['cluster']['mysql']['package']['archive'] = "#{node['cluster']['mysql']['package']['root']}/#{node['cluster']['mysql']['package']['platform']}/#{node['cluster']['mysql']['package']['file-name']}"
+default['cluster']['mysql']['package']['source'] = "#{node['cluster']['artifacts_s3_url']}/source/mysql-#{node['cluster']['mysql']['package']['source-version']}.tar.gz"
+
+# MySQL Validation
```

Comment:
If I understood correctly, with "MySQL Validation" you mean the expectations during kitchen tests.
If this the case, please specify that these are not installed but used in Kitchen tests.